lib/modules/datasource/dart/index.ts

```diff
@@ -23,7 +24,7 @@ export class DartDatasource extends Datasource {
       return null;
     }
     let result: ReleaseResult | null = null;
-    const pkgUrl = `${registryUrl}api/packages/${packageName}`;
+    const pkgUrl = joinUrlParts(registryUrl, `api/packages/${packageName}`);
```

Comment:
```suggestion
    const pkgUrl = joinUrlParts(registryUrl, 'api/packages', packageName);
```

we should be consistent